grub-mkconfig.md

```diff
@@ -0,0 +1,8 @@
+# grub-mkconfig
+
+> Generate a GRUB configuration file.
+> More information: <https://www.gnu.org/software/grub/manual/grub/html_node/Invoking-grub_002dmkconfig.html>
+
+- Generate the configuration:
+
+`grub-mkconfig -o /boot/grub/grub.cfg`
```

Comment:
I know, this may never change, but again, this is what we usually do.  It also just makes it clearer what files are being modified, etc.
```suggestion
`grub-mkconfig -o {{/boot/grub/grub.cfg}}`
```